app/main.py

```
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command_files: str) -> None:
+    parts = command_files.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    file_name_1, file_name_2 = parts[1], parts[2]
+    if f"{file_name_1}" == f"{file_name_2}":
+        return
+
+    try:
+        with (open(file_name_1, "r") as file_in,
+              open(file_name_2, "w") as file_out):
+            file_out.write(file_in.read())
+    except FileNotFoundError:
+        pass
```

This violates the checklist requirement: "Do not include auxiliary/debug code such as prints, comments for debugging, or extra helper functions in the final submission." and "Do not check for file existence; simply attempt to open the source file as required." You should remove the `try/except FileNotFoundError` block and let exceptions propagate naturally.